File: flutter-main/flutter-main/Inheritence/p11.dart
abstract class InterfaceDemo1{
	void m1(){
		print("In m1");
	}
}
abstract class InterfaceDemo2{
	void m2(){
		print("In m2");
	}
}
class Demo implements InterfaceDemo1,InterfaceDemo2{
	void m1(){
		print("In m1");
	}
	void m2(){
		print("In m2");
	}
}
void main(){
}

File: java-main/java-main/Nestedforloop/p11.java
/*   *    *
     *    *
     *    *
     *    * */class Demo{

	public static void main(String[]args){

		int N=4;
		for(int i=1;i<=4;i++){
			for(int j=1;j<=4;j++){
				if(j%N==1 || j%N==0){
					System.out.print("* ");
				}else{
					System.out.print("   ");
				}
			}
			System.out.println();
		}
	}
}
